### ISSUE TITLE
text report generation failed

### ISSUE THREAD

**Issue by ArnaudBuchholz** [ArnaudBuchholz/ui5-test-runner#62]
[job.zip](https://github.com/ArnaudBuchholz/ui5-test-runner/files/12703844/job.zip)

NOTE: this occurred because of a page timeout : 

```bash
ui5-test-runner --url https://ui5.sap.com/test-resources/sap/m/demokit/cart/webapp/test/integration/opaTestsComponent.qunit.html  --page-timeout 60s
```

**Comment by ArnaudBuchholz** [comment]
Whenever a test **times out**, the job report does not contain enough information to document the issue.
Since everything is based on pages :
* if a global timeout occurs, the page does not even exist in `qunitPages`
* if the timeout occurs within the page, no information is written

**Comment by ArnaudBuchholz** [comment]
Need to also handle `--fail-fast`

**Comment by ArnaudBuchholz** [comment]
fixed in 3.3.2
